code/modules/clothing/under/accessories/holster.dm

```diff
@@ -81,8 +82,11 @@
 		w_class = initial(w_class)
 		clear_holster()
 
-/obj/item/clothing/accessory/holster/attack_hand(mob/user)
-	if (has_suit) // If we are part of a suit.
+/obj/item/clothing/accessory/holster/attack_hand(mob/living/carbon/human/user)
```

Comment:
This isn't the proper way to do it, respect the function header and typecheck/cast as needed afterwards

Reply:
![tumblr_05be8c04fd48fca12d51bc32a83abcce_3c0b8f50_1280](https://github.com/Aurorastation/Aurora.3/assets/78785817/f47da659-435d-4d48-a19f-c6aec1872390)